src/main/java/uk/ac/ebi/eva/contigalias/controller/BaseController.java

```diff
@@ -31,6 +31,12 @@
 
     public static final int DEFAULT_PAGE_SIZE = 10;
 
+    public static final String API_PARAM_VALUE_PAGE_NUMBER = "Page number of required page. Page numbers start from 0" +
+            ". If not specified then default page number is " + DEFAULT_PAGE_NUMBER + ".";
```

Comment:
I'm getting these exceptions in the web service log:
```
2020-07-13 18:28:46.730  WARN 10670 --- [io-8080-exec-10] i.s.m.p.AbstractSerializableParameter    : Illegal DefaultValue null for parameter type integer

java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[na:1.8.0_211]
	at java.lang.Long.parseLong(Long.java:601) ~[na:1.8.0_211]
```
Maybe you can fix them by using defaultValue (`@ApiParam(value = API_PARAM_VALUE_PAGE_SIZE, defaultValue = "42 results per page")`) instead of putting it here in the param description in prose.